src/hooks/useRealtimeTranscription.ts

```
@@ -0,0 +1,286 @@
+        break;
+        
+      case 'conversation.item.input_audio_transcription.delta':
+        const delta = message.delta || '';
+        setState(prev => ({ 
+          ...prev, 
+          currentTranscription: prev.currentTranscription + delta 
+        }));
+        break;
+        
+      case 'conversation.item.input_audio_transcription.completed':
+        const transcript = message.transcript || '';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)